tests/basic/upgrade_test.go

```diff
@@ -143,8 +139,12 @@ var _ = Describe("{UpgradeVolumeDriver}", func() {
 
 			// Perform upgrade hops of volume driver based on a given list of upgradeEndpoints passed
 			for _, upgradeHop := range strings.Split(Inst().UpgradeStorageDriverEndpointList, ",") {
+				currPXVersion, err := Inst().V.GetDriverVersionOnNode(storageNodes[0])
```

Comment:
storageNodes[0] also need to be updated for every hop

Reply:
fixed